libs/design/src/molecules/modal/modal.md

```diff
@@ -24,7 +27,7 @@ export class ModalModule {
 }
 ```
 
-Once your module is setup, you can take advantage of the `DaffModalService` that is provided by the `DaffModalModule` to open and close your instance of the `DaffModalComponent`. 
+Once you have configured your `@NgModule` you can take advantage of the `DaffModalService` that is provided by the `DaffModalModule` to `open` and `close` the instance of the `DaffModalComponent`. 
```

Comment:
Once you have configured **the** `@NgModule`**,** you can take advantage of the ......

Reply:
I feel "your" is more appropriate given that the user is configuring their own particular module.

Reply:
Once your `@NgModule` has been configured, ....